### PR TITLE
style: Remove border from Reveal in example page

### PR DESCRIPTION
**Pull request by greenbech**
#### Type of change
Style 

#### Jira ticket :blue_book:
No JIRA ticket

## Description :pencil:
This PR removes the glaring white border from Reveal on the example page. After this PR, the page looks like this:

Skjermbilde 2022-12-14 kl  11 18 36

Since the axis view controller (not exactly sure what to call this) is on the bottom right _and_ the FPS now overlaps the Reveal container more, I decided to move the FPS meter to the bottom left instead. 

## How has this been tested? :mag:

By running the example page and opening `localhost:3000`. 

## Test instructions :information_source:

Run the example page locally and verify that the styling looks correct. More concretely I would look for:

- Does the Reveal container fill up the entire screen?
- Is some of the Reveal containers outside of the browser's viewport? (e.g. does scroll bars appear or similar)

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [x] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [x] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [x] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: examples/public/index.html [hunk 1]
```diff
@@ -24,8 +24,11 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Reveal examples</title>
-  <script defer src="/bundle.js"></script><script defer src="/bundle.js"></script><script defer src="/bundle.js"></script></head>
-  <body>
+    <script defer src="/bundle.js"></script>
+    <script defer src="/bundle.js"></script>
+    <script defer src="/bundle.js"></script>
```
Comment (greenbech) [inline examples/public/index.html:29]: Is there any reason this is imported trice? (The formatting was done automatically by my prettier extension)

File: examples/src/index.tsx [hunk 2]
```diff
@@ -2,8 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 
-import './index.css';
-
```
Comment (greenbech) [inline examples/src/index.tsx:6]: I decided to move this inline in the `index.html` page to remove some complexity

**Review by haakonflatval-cognite — APPROVED** [review]
Okay, it actually looks a lot better now.. Thanks :+1:

File: examples/src/index.tsx (hunk 2 above)
Comment (haakonflatval-cognite) [reply examples/src/index.tsx:6]: Sounds good :+1:

**Pull request merged**